ccpp/physics_namelists/input_GFS_v15p2.nml

```diff
@@ -14,6 +14,8 @@
        iems           = 1
        iaer           = 111
        ico2           = 2
+       iovr_sw        = 5
+       iovr_lw        = 5
```

Comment:
I am not sure what is the default of iovr_sw and _lw, but we do not want to change the behavior of this suite. It should produce the same results as before